src/boot.lisp

```diff
@@ -409,66 +411,126 @@
     `(multiple-value-call (lambda ,gvars ,@setqs)
        ,@form)))
 
+;;; types utils
+(defun object-type-code (object) (oget object "dt_Name"))
```

Comment:
What do `dt_` stand for?

Reply:
This is the prototype of the tag system. Today I do not have a final vision of the image of this system.

So far like this.

Finally, it will be possible to work on this after the implementation of the` structure` and some possible changes in `clos`.